playbooks/portals-setup-following.yml

```diff
@@ -2,42 +2,71 @@
   hosts: webportals
   remote_user: "{{ webportal_user }}"
   gather_facts: True
-  serial: 1              # Limit concurrency
+  serial: 1 # Limit concurrency
```

Comment:
Follow Up: could we experiment with parallel `host_pinned` setups? that would help speed up server setups I would think.

Reply:
`host_pinned` startegy is a bit dangerous:

`[WARNING]: Using any_errors_fatal with the free strategy is not supported, as tasks are executed independently on each host`

In practice when we want to update a env variable on all prod portals and the `portals-setup-following` playbook contains an error (new error is introduced or new corner case is hit) we shutdown the complete cluster.

We are however free to execute the playbook in parallel in parallel shell sessions.